config/webpack.config.dev.js

```diff
@@ -143,6 +143,15 @@ module.exports = {
           limit: 10000,
           name: 'static/media/[name].[ext]'
         }
+      },
+      // "html" loader is used to process template page (index.html) to resolve
+      // resources linked with <link href="./relative/path"> HTML tags.
+      {
+        test: /\.html/,
+        loader: 'html',
+        query: {
+          attrs: ['link:href'],
```

Comment:
I wouldn't bother adding more at the moment but rather serve on per-request basis.